### PR TITLE
Ember press section

### PR DESCRIPTION
**Pull request by BradLeftley**

### CODE REVIEW

File: source/2019-03-30-the-ember-times-issue-91.md [hunk 1]
```diff
@@ -53,8 +53,15 @@ This video is a nice introduction into the world of Ember Octane and Glimmer com
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Empress: the Ember Press Episode 129](https://emberweekend.com/episodes/empress-the-ember-press)
+
+Episode 129 of the Ember press featured [@mansona](https://github.com/mansona) the creator of Authmaker. This episode included a range of topics including the migration of moving the Ember guides from Middle man to be powered by Ember. Other areas include Middleman shortcomings and the rework of the guides.
```
Comment (MinThaMie) [inline source/2019-03-30-the-ember-times-issue-91.md]: I think it is episode 129 of the EmberWeekend podcast, and the subject is Ember Press/Empress.
Secondly, you write Middle man with and without space between them :)

File: source/2019-03-30-the-ember-times-issue-91.md [hunk 2]
```diff
@@ -53,8 +53,15 @@ This video is a nice introduction into the world of Ember Octane and Glimmer com
...
+
+[@mansona](https://github.com/mansona) also talks about his journey using Broccoli Static JSON to convert markdown into JSON API. The ultimate goal was to use the Broccoli pipeline markdown to convert to JSON API and use static files!
+
+[@mansona](https://github.com/mansona) also introduced Guidemaker:  A fully functional, static site implementation of a documentation site and EmberJS with fully out of the box SEO friendly output! With support to be hosted on S3 or any other site hosting solution. Check it out [here](https://github.com/empress/guidemaker).
 
+Listen to the full podcast [here]( https://emberweekend.com/episodes/empress-the-ember-press).
```
Comment (MinThaMie) [inline source/2019-03-30-the-ember-times-issue-91.md]: I learned from my first blogpost that it is not really accessible to do the link on the `here`. It is better to do it on the `podcast` to make it more explicit where the link goes to :)

**Review by MinThaMie — CHANGES_REQUESTED** [review]
Cool that you wrote your first piece, I did found some things you could look into. Hopefully it helps!

File: source/2019-03-30-the-ember-times-issue-91.md (hunk 1 above)
Comment (jayjayjpg) [inline source/2019-03-30-the-ember-times-issue-91.md]: We use a language linting tool on the blog called [alex](https://github.com/get-alex/alex) which helps us to catch insensitive language during our CI run with Travis. Currently this PR fails the language check with the following error message:

```
source/2019-03-30-the-ember-times-issue-91.md
  58:257-58:266  warning  `Middleman` may be insensitive, use `Intermediary`, `Go-between` instead  middleman-middlewoman  retext-equality
```
You can also find the [full log over here](https://travis-ci.org/ember-learn/ember-blog/builds/512893636?utm_source=github_status&utm_medium=notification).

You should be able to allow the usage of the term `Middleman` by applying an exception rule. You can read more about it [in our contribution guides as well](https://github.com/ember-learn/ember-blog/blob/master/source/CONTRIBUTING.md#language-checking-with-alex)

```suggestion
<!--alex ignore middleman-middlewoman-->
Episode 129 of the Ember press featured [@mansona](https://github.com/mansona) the creator of Authmaker. This episode included a range of topics including the migration of moving the Ember guides from Middleman to be powered by Ember. Other areas include Middleman shortcomings and the rework of the guides.
```

**Review by jayjayjpg — APPROVED** [review]
This reads great! Thank you for this :sparkles: writeup this week @BradLeftley and thank you for the great chief review @MinThaMie!

**Pull request merged**